cg/meta/deliver/utils.py

```diff
@@ -92,12 +94,13 @@ def should_include_file_case(file: File, sample_ids: set[str], workflow: str) ->
     return any(tags.issubset(tags_on_file) for tags in case_tags)
 
 
-def should_include_file_sample(file: File, workflow: str) -> bool:
+def should_include_file_sample(file: File, workflow: str, sample_internal_id: str) -> bool:
     """Check if file should be included in sample bundle.
 
     At least one tag should match between file and the sample tags.
     """
     file_tags = {tag.name for tag in file.tags}
+    file_tags.add(sample_internal_id)
```

Comment:
Should we not exclude this tag for fastq deliveries? Just comparing with https://github.com/Clinical-Genomics/cg/blob/5872f73334059826529f056395831b9ca33070ff/cg/meta/deliver.py#L287-L305

Reply:
Oooo good thought! 😎